### ISSUE TITLE
Add an 'alt' attribute to <img> elements

### ISSUE THREAD

**Issue by SchoolCitizenAssemblies** [SchoolCitizenAssemblies/SCA#9]
Helps anyone who cannot load images or for screen readers
